### PR TITLE
Add templates for Zenfolio Development Teams

### PR DESCRIPTION
**Pull request by chris-zenfolio**
Adding 2 templates for Zenfolio Development teams to use for testing

### CODE REVIEW

File: zenfolio-test.customdomain,json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "providerId": "Zenfolio",
+    "serviceId": "CustomDomain-Test",
```
Comment (pawel-kow) [inline zenfolio-test.customdomain,json:3]: this file name has comma instead of dot in the name.

File: zenfolio-test.customsubdomain.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "providerId": "Zenfolio",
```
Comment (pawel-kow) [inline zenfolio-test.customsubdomain.json:2]: AssertionError: File name zenfolio-test.customsubdomain.json not matching the pattern. Expected zenfolio.customsubdomain-test.json

**Review by pawel-kow — CHANGES_REQUESTED** [review]

**Comment by chris-zenfolio** [comment]
Filename corrected.  Apologies for the typo.

File: zenfolio-test.customdomain,json (hunk 1 above)
Comment (chris-zenfolio) [reply zenfolio-test.customdomain,json]: Corrected the filename and committed/pushed the update to the branch.

File: zenfolio-test.customsubdomain.json (hunk 2 above)
Comment (chris-zenfolio) [reply zenfolio-test.customsubdomain.json]: file was renamed to match expected pattern.

File: zenfolio-test.customdomain,json (hunk 1 above)
Comment (chris-zenfolio) [reply zenfolio-test.customdomain,json]: file was also renamed to match expected pattern.

**Review by pawel-kow — APPROVED** [review]

**Pull request merged**